### DISCUSSION TITLE
How to calculate recommended polygons from the device performance?

### DISCUSSION THREAD

**Discussion by JMAR09** [google/model-viewer#4000] [Q&A]
Hi all, 

I know that is a tricky question but it would be great to know it. Is there any easy way how to calculate recommended polygons from device performance? It would be awesome to calculate how many polygons can handle for example iPhone 7 or some old Android.

Thank you very much

Have a nice day :) 

Jan

**Comment by elalish** [comment]
No, it's way more complicated than that. Honestly draw calls and texture density probably have even more effect than polygons, and they all interact. Testing is your friend.
